docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -1092,6 +1092,9 @@
                   },
                   "/rels/rendition_type/2048": {
                     "href": "assets/<cover_asset_id>/revisions/<revision_id>/renditions/<rendition_id>"
+                  },
+                  "/rels/xmp/develop": {
+                    "href": "assets/<cover_asset_id>"
```

Comment:
1. href should end in /xmp/develop
2. fileSize and invalid also need to be added